attractions/snackbar/snackbar-container.svelte

```diff
@@ -1,9 +1,23 @@
 <script>
+  /**
+   * @slot {{ showSnackbar: (options: {
+   *   component: typeof import("svelte").SvelteComponentDev,
+   *   props: Record<string, any>,
+   *   duration: number,
+   * }) => {
+   *   close: () => void,
+   *   expired: Promise<boolean>,
+   * } }}
+   */
   import { setContext } from 'svelte';
   import Snackbar from './snackbar.svelte';
   import snackbarContextKey from './snackbar-context-key.js';
   import SnackbarPositions from './snackbar-positions.js';
 
+  /**
+   * The position of the snackbar stack inside the window.
+   * @type {typeof import('../../snackbar/snackbar-positions').default}
+   */
   export let position = SnackbarPositions.BOTTOM_LEFT;
   let registeredSnackbars = new Set();
```

Comment:
Can't comment on the relevant line, but I guess the `showSnackbar` function also needs to be documented as the DropdownShell's `toggle`.

Reply:
It is documented in the top as a slot prop. `export function ...` syntax is not currently being handled correctly by sveld; it detects it as a normal prop whose default value is that function.

Reply:
Yeah, but it's also exposed as a regular prop. In `dropdown-shell.svelte` you added a comment to the exported function, regardless of how sveld supports it, I mean we should add a similar comment here

Reply:
I did, but it's not like it matters. It still didn't give it the correct type. It might be worth it only for the description (for the time being, until sveld fixes the issue)